model/src/main/java/com/chiralbehaviors/CoRE/network/NetworkAuthorization.java

```diff
@@ -39,6 +39,16 @@
 /**
  *
  * The abstract super class of all network authorizations.
+ * 
+ * from left to right: classification, classifier, auth parent, auth
+ * relationship, child relationship, cardinality
+ * 
+ * {classification, classifier} is the “parent”
```

Comment:
So, you might want to reverse these to be in line with the auth parent, auth relationship.  Or vice versa.  I think it makes sense to say:  isA Thing rather than Thing isA.  Sounds like yoda.